tools/test/audio/asrc_run.sh

```diff
@@ -11,4 +11,4 @@ fi
 COMP=asrc
 DIRECTION=playback
 
-./comp_run.sh $COMP $DIRECTION $1 $2 $3 $4 $5 $6
+./comp_run.sh $COMP $DIRECTION "$1" "$2" "$3" "$4" "$5" "$6"
```

Comment:
Have you considered "$@" ?
https://www.gnu.org/software/bash/manual/html_node/Special-Parameters.html

same comment below

Reply:
Yep, that's better

Reply:
Done